plugins/outputs/influxdb_v2/influxdb_v2_test.go

```diff
@@ -906,3 +906,261 @@ func TestUseDynamicSecret(t *testing.T) {
 	require.NoError(t, secretToken.Set([]byte(token)))
 	require.NoError(t, plugin.Write(metrics))
 }
+
+func BenchmarkWrite1k(b *testing.B) {
+	batchsize := 1000
+
+	// Setup a test server
+	ts := httptest.NewServer(
+		http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
+			w.WriteHeader(http.StatusNoContent)
+		}),
+	)
+	defer ts.Close()
+
+	// Setup plugin and connect
+	plugin := &influxdb.InfluxDB{
+		URLs:   []string{"http://" + ts.Listener.Addr().String()},
+		Token:  config.NewSecret([]byte("sometoken")),
+		Bucket: "my_bucket",
+		Log:    &testutil.Logger{},
+	}
+	require.NoError(b, plugin.Init())
+	require.NoError(b, plugin.Connect())
+	defer plugin.Close()
+
+	metrics := make([]telegraf.Metric, 0, batchsize)
+	for i := range batchsize {
+		metrics = append(metrics, metric.New(
+			"cpu",
+			map[string]string{
+				"bucket": "foo",
+			},
+			map[string]interface{}{
+				"value": float64(i),
+			},
+			time.Unix(0, 0),
+		))
+	}
+
+	// Benchmark the writing
```

Comment:
Reset the timer to exclude setup time.
```suggestion
	// Benchmark the writing
	b.ResetTimer()
```